cmd/output/csv.go

```diff
@@ -0,0 +1,37 @@
+package output
+
+import (
+	"strconv"
+
+	"github.com/flanksource/canary-checker/pkg"
+	"github.com/jszwec/csvutil"
+)
+
+type CSVResult struct {
+	CheckType   string `csv:"checkType"`
+	Pass        bool   `csv:"pass"`
+	Time        string `csv:"time"`
```

Comment:
```suggestion
	Duration        string `csv:"duration"`
```

Can we also add namespace, name, and endpoint ?